include/lsst/afw/geom/SkyWcs.h

```diff
@@ -405,6 +408,7 @@ class SkyWcs final : public table::io::PersistableFacade<SkyWcs>, public table::
      */
     std::shared_ptr<ast::FrameDict> _checkFrameDict(ast::FrameDict const &frameDict) const;
 
+    std::shared_ptr<const ast::FrameDict> _frameDict;
     std::shared_ptr<const TransformPoint2ToSpherePoint> _transform;
```

Comment:
It's worth having a comment here that explains the relationship between `_frameDict` and `_transform`.